src/topNavigation/components/favouritesIcon.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { IconHeart } from 'hds-react';
+
+interface Props {
+  count: number;
+}
+
+const FavouritesIcon = (props: Props): JSX.Element => {
+  return (
+    <div className="TopNavigation_FavouritesIcon">
```

Comment:
Top element of the component, so should only have the block part. Strictly speaking, should be just `FavouritesIcon`, but `TopNavigationFavouritesIcon` would be more descriptive so I'd use that.